src/tsickle.ts

```diff
@@ -920,6 +925,9 @@ class Annotator extends ClosureRewriter {
             if (sym.flags & ts.SymbolFlags.Alias) {
               sym = typeChecker.getAliasedSymbol(sym);
             }
+            if (this.newTypeTranslator(heritage).isBlackListed(sym)) {
```

Comment:
Is it important that this use a different context parameter than all the other uses of typetranslator?